internal/facilities/retrieve.go

```diff
@@ -38,18 +39,21 @@ func (c *Client) Retrieve() *cobra.Command {
 
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-			facilities, _, err := c.Service.List(c.Servicer.ListOptions(nil, nil))
+			inc := []string{}
+			exc := []string{}
+			facilitiesList, _, err := c.Service.FindFacilities(context.Background()).Include(inc).Exclude(exc).Execute()
```

Comment:
This needs to be wired up to the `Includes()` and `Excludes()` functions that you added earlier

Reply:
updated